### PR TITLE
Update quick installer example for v2 config

### PR DESCRIPTION
**Pull request by smunilla**
Update of the quick installer instructions to match the new arbitrary variables config file.

### CODE REVIEW

File: install_config/install/quick_install.adoc [hunk 1]
```diff
@@ -142,44 +142,53 @@ xref:running-an-unattended-installation[unattended installation].
...
+      - node
+    connect_to: 10.0.0.3
+  roles: <12>
+    master:
+      variable: "value" <13>
+    node:
+      variable: "value" <13>
 ----
 <1> The version of this installation configuration file. As of {product-title} 3.1,
 the only valid version here is `v1`.
```
Comment (adellape) [inline install_config/install/quick_install.adoc]: Description needs an update probably?

**Comment by smunilla** [comment]
@adellape How's this look? The footnotes wording could probably be better I think.

File: install_config/install/quick_install.adoc [hunk 2]
```diff
@@ -201,6 +210,9 @@ Containerized] for more details.
 upgrading, or uninstalling the systems. If the configuration file was
 auto-generated, then this is the value you first enter for the host during that
 interactive install process.
+<12> Node labels can optionally be set per-host.
+<13> Defines a dictionary of roles across the deployment.
+<14> Any ansible variables that should only be applied to hosts assigned a role can be defined. For examples, see https://docs.openshift.com/enterprise/latest/install_config/install/advanced_install.html#configuring-host-variables
```
Comment (adellape) [inline install_config/install/quick_install.adoc]:
```
For examples, see xref:../../install_config/install/advanced_install.adoc#configuring-ansible[Configuring Ansible].
```

File: install_config/install/quick_install.adoc [hunk 3]
```diff
@@ -142,44 +142,54 @@ xref:running-an-unattended-installation[unattended installation].
...
+  - ip: 10.0.0.3
+    hostname: node2-private.example.com
+    public_ip: 24.222.0.3
+    public_hostname: node2.example.com
+    roles:
+      - node
+    connect_to: 10.0.0.3
+  roles: <13>
+    master:
+      variable: "value" <14>
```
Comment (adellape) [inline install_config/install/quick_install.adoc]: Maybe to show the form of "value":

```
  roles: <13>
    master:
      variable: "<variable_name1>=<value1>" <14>
    node:
      variable: "<variable_name2>=<value2>, <variable_name3>=<value3>" <14>
```

^ Does it take a comma-separated list?

Comment (adellape) [inline install_config/install/quick_install.adoc]: Or should it be:

```
  roles: <13>
    master:
      <variable_name1>: "<value1>"
[...]
```

Comment (smunilla) [inline install_config/install/quick_install.adoc]: It's a YAML dictionary, so probably the second one. Will update.

File: install_config/install/quick_install.adoc [hunk 4]
```diff
@@ -201,6 +211,10 @@ Containerized] for more details.
 upgrading, or uninstalling the systems. If the configuration file was
 auto-generated, then this is the value you first enter for the host during that
 interactive install process.
+<12> Node labels can optionally be set per-host.
+<13> Defines a dictionary of roles across the deployment.
+<14> Any ansible variables that should only be applied to hosts assigned a role can be defined.
+For examples, see For examples, see xref:../../install_config/install/advanced_install.adoc#configuring-ansible[Configuring Ansible].
```
Comment (adellape) [inline install_config/install/quick_install.adoc]: Oops, double "For examples"

**Comment by adellape** [comment]
:+1:

**Comment by vikram-redhat** [comment]
@ahardin-rh - can you please follow up on this early next week?

**Comment by adellape** [comment]
@ahardin-rh @smunilla I think whatever follow-up edits I had in mind, Sam and I had already discussed over IRC. Either that or I can't remember what I was thinking! So, removing `to_followup` label.

**Pull request merged**